File: problem_forty_three.js
//Pascal's triangle
var value = 10
function MyFunction(value){
    var list =[]
    for(var i = 0;i<value;i++){
        var arr =[]
        if (i == 0){
            console.log(1)
            console.log(1,1)
            list.push(1)
            list.push(1)
        }
        else {
            for (var j = 0; j<list.length-1;j++){
                if(j ==0){
                    arr.push (1)
                    var val= list[j]+list[j+1]
                    arr.push(val)
                }else{
                    var valu=list[j]+list[j+1]
                    arr.push(valu)
                }
            }
            arr.push(1)
            for (var k=0; k<arr.length;k++){
                console.log(arr[k])
            }
            list =arr
        }
        
    
    }
    
}
MyFunction(value)

File: problem_forty_three.py
#Pascal's triangle
inp = int(input("Enter the number:"))
def MyFunction(inp):
    lst =[]
    for i in range(inp): 
        lst2 =[]
        if i == 0:
            print(1)
            print(1,1)    
        elif i == 1:
            print(1,i+1,1)
            lst.append(1)
            lst.append(i+1)
            lst.append(1)
        elif i>1:
            for j in range(len(lst)-1):
                if j == 0:
                    lst2.append(1)
                    value = lst[j]+lst[j+1]
                    lst2.append(value)
                elif j >0:
                    val =lst[j]+lst[j+1]   
                    lst2.append(val)  
            lst2.append(1)
            listToStr = ' '.join([str(elem) for elem in lst2]) 
            print(listToStr)
            lst =lst2                
MyFunction(inp)









               

         


